src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs

```diff
@@ -68,30 +71,30 @@ public async Task<FluentHttpResponse> Invoke(FluentHttpMiddlewareContext context
 				&& !options.ShouldLogDetailedResponse.GetValueOrDefault(false))
 			{
 				response = await _next(context);
-				_logger.LoggerHttp_CondensedRequest(request.Method, request.Uri!, response.StatusCode, watch.GetElapsedTime().TotalMilliseconds);
+				_logger.LoggerHttp_CondensedRequest(request.Method, request.Uri!, path, response.StatusCode, watch.GetElapsedTime().TotalMilliseconds, httpVersion);
 				return response;
 			}
 
 			if (!(options.ShouldLogDetailedRequest ?? false))
-				_logger.LoggerHttp_Request(request);
+				_logger.LoggerHttp_Request(request.Method, request.Uri!, path, httpVersion);
 			else
 			{
 				string? requestContent = null;
 				if (request.Message.Content != null)
 					requestContent = await request.Message.Content.ReadAsStringAsync();
-				_logger.LoggerHttp_RequestDetailed(request, request.Headers.ToFormattedString(), requestContent);
+				_logger.LoggerHttp_RequestDetailed(request.Method, request.Uri!, path, request.Headers.ToFormattedString(), requestContent, httpVersion);
 			}
 
 			response = await _next(context);
 			var stopwatchElapsed = watch.GetElapsedTime();
 			if (response.Content == null || !(options.ShouldLogDetailedResponse ?? false))
 			{
-				_logger.LoggerHttp_Response(response, stopwatchElapsed.TotalMilliseconds);
+				_logger.LoggerHttp_Response(request.Method, request.Uri!, path, response.StatusCode, stopwatchElapsed.TotalMilliseconds, httpVersion);
 				return response;
 			}
 
 			var responseContent = await response.Content.ReadAsStringAsync();
-			_logger.LoggerHttp_ResponseDetailed(response, response.Headers.ToFormattedString(), responseContent, stopwatchElapsed.TotalMilliseconds);
+			_logger.LoggerHttp_ResponseDetailed(request.Method, request.Uri!, path, response.StatusCode, response.Headers.ToFormattedString(), responseContent, stopwatchElapsed.TotalMilliseconds, httpVersion);
```

Comment:
httpVersion theres one in the request and one in the response - for this one we can log the response one

Reply:
done